runtime/doc/ui.txt

```diff
@@ -796,6 +796,7 @@ must handle.
 		"echo"		|:echo| message
 		"echomsg"	|:echomsg| message
 		"echoerr"	|:echoerr| message
+		"insert_expand" |insert_expand| message
```

Comment:
`:help insert_expand` seems to be about CTRL-X. Should we name this "completion" and use it for any other potential completion-related messages (e.g. for ctrl-n)?
```suggestion
		"completion" |ins-completion| message
```

Reply:
Ye seems more appropriate to me, if not `"completion_menu"`. The `'shortmess'` flag that guards these messages links to `|ins-completion-menu|`, and we only emit these as actual messages with `:set noshowmode`, both of which seem good to mention in this help text.